fe4u-boilerplate-main/src/index.html

```diff
@@ -1,20 +1,375 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html lang="eng">
 
-   <head>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="msapplication-tap-highlight" content="no">
 
-      <meta charset="utf-8">
-      <meta http-equiv="x-ua-compatible" content="ie=edge">
-      <meta name="msapplication-tap-highlight" content="no">
-      <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Teachinder</title>
 
-      <title>Project</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="уникальный сайт учителей с разных стран мира">
+    <meta name="keywords" content="Teacher,Site,TopTeacher,Teachinder,learn,web course">
+    <meta name="author" content="Igor Kharlamov">
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.8.0/dist/leaflet.css"
+          integrity="sha512-hoalWLoI8r4UszCkZ5kL8vayOGVae1oxXe/2A4AO6J9+580uKHDO3JdHb7NzwwzK5xr/Fs0W40kiNHxM9vyTtQ=="
+          crossorigin=""/>
+    <link rel="icon" type="image/x-icon" href="./favicon.ico">
+    <link rel="stylesheet" type="text/css" href="css/avatar-wrapper.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://kit.fontawesome.com/88c44a0b03.js" crossorigin="anonymous"></script>
```

Comment:
This way you block the page loading until JS is loaded. You should do the next:
- add all scripts at the end of the html content
- if adding at the end isn't possible, set `async` or `defer` attribute for every <script> https://learn.javascript.ru/script-async-defer